src/backend/ai-reviewer-api/src/main/resources/logback-spring.xml

```diff
@@ -29,16 +29,17 @@
                     <pattern>%msg</pattern>
                 </layout>
             </appender>
+            
+            <!-- LOG everything at INFO level -->
+            <root level="info">
+                <appender-ref ref="Console"/>
+                <appender-ref ref="splunk"/>
+            </root>
         </then>
+        <else>
+            <root level="info">
+                <appender-ref ref="Console"/>
+            </root>
+        </else>
     </if>
-
-    <!-- LOG everything at INFO level -->
-    <root level="info">
-        <appender-ref ref="Console"/>
-        <if condition='isDefined("SPLUNK_URL")'>
```

Comment:
An if element is not valid inside a root element. The new version of spring enforces correct xml or logging won't work at all.